backend/api/models.py

```diff
@@ -13,18 +17,22 @@ class CoursePlan(models.Model):
     startAt = models.DateField()
     endAt = models.DateField()
     # even week: True, odd week: False
-    WEEKTYPE = [(True, 'EVEN'), (False, 'ODD')]
     firstWeekType = models.BooleanField(choices=WEEKTYPE)
 
+    def __str__(self):
+        return '{}, {}, {}, {}'.format(self.faculty, self.course, self.studiesMode, self.term)
```

Comment:
Use f-string. It looks much better and it is shorter:
`return f"{self.faculty}, {self.course}, {self.studiesMode}, {self.term}"`